src/lib/exports.ts

```diff
@@ -67,10 +67,12 @@ export async function convertNoteDocToHtmlString(
     .use(rehypeKatex)
     .process(note.content)
 
-  const [htmlString, attachmentUrls] = await updateNoteLinks(
+  const attachmentUrls: string[] = []
```

Comment:
Revert this. I don't see any benefits to predefine this array.

Reply:
Removed!